code/modules/defenses/defenses.dm

```diff
@@ -292,6 +381,7 @@
 
 	if(health <= 0 && stat != DEFENSE_DESTROYED)
 		stat |= DEFENSE_DESTROYED
+		SEND_SIGNAL(src, COMSIG_SENTRY_DESTROYED_ALERT, src)
```

Comment:
sending src as an argument is uncessary, all signals send `datum/source` implicitly